src/helpers/sql.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Query\Builder;
+if (!function_exists('dumpSql')) {
+    /**
+     * Print SQL query with bindings.
+     *
+     * @param Builder $builder
+     * @return string
+     */
+    function dumpSql(Builder $builder)
```

Comment:
All other functions in this project use snake case. Additionally, this file should be renamed to `dump_sql`. This is so the service provider can check if the function exists.